File: primeNumber.py
def primeNumber(num):
    if (num == (0 or 1)):
        print(str(num) + ' is NOT a prime number')
    elif (num != 0 and num % num == 0):
        print(str(num) + ' is a prime number')
    else:
        print(str(num) + ' is NOT a prime number')
    
# test cases
primeNumber(2)
primeNumber(3)
primeNumber(43)
primeNumber(0)
primeNumber(1)

File: primeNumber.js
function primeNumber(num) {
  if (num == (1 || 0)) {
    return `${num} is NOT a prime number`;
  } else if (num % num == 0) {
    return `${num} is a prime number`;
  } else {
    return `${num} is NOT a prime number`;
  }
}

//test cases
console.log(primeNumber(2));
console.log(primeNumber(3));
console.log(primeNumber(43));
console.log(primeNumber(1));
console.log(primeNumber(0));
